docs/metrics/lead-cycle-time.md

```diff
@@ -0,0 +1,33 @@
+# Lead and cycle time metrics
+
+Monitoring you team's lead time and cycle time allows you to understand if you're improving the ability to deliver value to customers.
+
+![Lead time versus cycle time](images/lead-cycle-time.png)
+
+-   [Lead time](#lead-time)
+-   [Cycle time](#cycle-time)
+
+!!! note
+    Pulse calculates lead time and cycle time based on the state changes of issues **that are already closed** in Jira. This means that:
+
+    -   **If you reopen** a completed issue, Pulse stops considering the issue while calculating the metrics. If the issue is completed again, Pulse takes into account the last completed timestamp.
+
+    -   **If you change the type or the project** of a completed issue, Pulse reflects the change on the dashboards, transitioning the issue history to the new project if relevant
+    
+        Note that this change doesn't affect the lead time or cycle time metrics.
```

Comment:
it might if a user is using a filter, no?

Reply:
Yeah that's what I meant. If you are using a filter the metric might be affected, but without filters the metric is never affected.

Reply:
I tweaked the sentence to clarify this.